### DISCUSSION TITLE
Test Blazor FluentUi

### DISCUSSION THREAD

**Discussion by tpurschke** [CactuseSecurity/firewall-orchestrator#2094] [Ideas]
https://www.fluentui-blazor.net/

**Comment by NilsPur** [comment]
+ DataGrid Component could be used to solve filtering problem: #1044 
+ Custom search Dropbox could be replaced in the future (multiselect currently not working / supported) -> Combobox
+ A lot of useful components and icons included
- A lot of work replacing the current components
